docs/reference/project_template.md

```diff
@@ -97,6 +98,24 @@ The Django settings files are split up into `base.py`, `dev.py`, `production.py`
 On production servers, we recommend that you only store secrets in ``local.py`` (such as API keys and passwords). This can save you headaches in the future if you are ever trying to debug why a server is behaving badly. If you are using multiple servers which need different settings then we recommend that you create a different ``production.py`` file for each one.
 ```
 
+### Running and Writing Tests
+
+When you create a new project using `wagtail start`, there will be a set of basic tests included in the `home/tests.py` file.
+
+These tests check:
+
+-   If the _root `Page`_ (ID=1) is automatically created.
+-   If the _home `Page`_ is created as a child of the root page.
+-   If a `BlogIndexPage` (or other page types) can be added.
```

Comment:
```suggestion
-   A placeholder test for other sub-pages, such as the `BlogIndexPage`, ready for implementation.
```

This test is not actually implemented at the start so we should be clear about this.

**Update** - maybe we just remove this placeholder test, which just fails, to avoid confusion.